layout.tmpl

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>iTowns API : <?js= title ?></title>
+
+    <script src="scripts/prettify/prettify.js"> </script>
+    <script src="scripts/prettify/lang-css.js"> </script>
+    <!--[if lt IE 9]>
+      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+    
+    <link type="text/css" rel="stylesheet" href="styles/jsdoc.css">
+    <link type="text/css" rel="stylesheet" href="styles/prettify.css">
+    <link type="text/css" rel="stylesheet" href="../doc.css">
```

Comment:
This doesn't work in the current form. `doc.css` never get copied to out/API_Doc. You need 3 things:

1. change jsdoc-config.json to: 
```js
{
    "templates": {
        "default": {
            "layoutFile": "layout.tmpl",
            "staticFiles": {
                "paths": ["static_jsdoc"]
           }
        }
    }
} 
 
```
2. move `./doc.css` into a subfolder of itowns, like `static_jsdoc` for instance (see 1.)

1 and 2 are needed because of  what happens here: https://github.com/clenemt/docdash/blob/master/publish.js#L503. This code fails for single files directly on the root folder of the git repo: on line 518, sourcePath would be `.`, outDir is whatever outDir you gave, and filename would be doc.css, so `filename.replace(...)` yields something like docout/ if outdir is out... (it's a bug in jsdoc/docdash imho)

3. change the href here to `./doc.css`